precompiles/ArbWasm.go

```diff
@@ -133,6 +135,9 @@ func (con ArbWasm) MinInitGas(c ctx, _ mech) (uint64, uint64, error) {
 	params, err := c.State.Programs().Params()
 	init := uint64(params.MinInitGas) * programs.MinInitGasUnits
 	cached := uint64(params.MinCachedInitGas) * programs.MinCachedGasUnits
+	if c.State.ArbOSVersion() < gethparams.ArbosVersion_StylusActivationFix {
+		return 0, 0, vm.ErrExecutionReverted
```

Comment:
I think this may differ in gas left, and it should probably only happen if `init > math.MaxUint8 || cached > math.MaxUint8`

Reply:
I don't think it does. see: https://github.com/OffchainLabs/nitro/pull/2655/

Reply:
my assumption is this is what was hit, which seems to keep gasLeft

```go
	encoded, err := method.template.Outputs.PackValues(result)
	if err != nil {
		log.Error("could not encode precompile result", "err", err)
		return nil, callerCtx.gasLeft, vm.ErrExecutionReverted
	}
	```

Reply:
(or if you think that test is missing something - LMK)

Reply:
Ah, I see. Nevermind, you're right it doesn't use any more gas than is used at that point. Still, I think we should add the check for if the return values exceed the size of uint8s.

Reply:
agreed, but that's a different PR (which will make the ArbDebug PR impossible)